app/assets/javascripts/application.js

```diff
@@ -1,4 +1,29 @@
 $(document).ready(function(){
 
+  function Task(params){
+    if(params) this.data = params;
+    this.render();
+  }
+
+  Task.prototype = {
+    render: function(){
+    }
+
+  function List(params, tasksHtml){
+    if(params){
+      this.data = params;
+      this.tasksHtml = tasksHtml;
+      this.render();
+    }
+  }
+
+  List.prototype = {
+    render: function(){
+    }
+  }
+
```

Comment:
You're off to a good start! Remember to write out pseudo code when you get stuck. I'd like to go over this in our one on one next week.